src/packages/sync-from-notion-db/index.ts

```diff
@@ -0,0 +1,28 @@
+import { PartialPageObjectResponse } from "@notionhq/client/build/src/api-endpoints";
+import { Env } from "../..";
+import { createNotionClient } from "./utils";
+
+export async function runSyncFromNotionDataBase(
+  env: Env
+): Promise<PartialPageObjectResponse[]> {
+  const notion = createNotionClient({
+    token: env.NOTION_TOKEN,
+  });
+  const libraries = await notion.queryPublishRoadmapLibraries();
+
+  if (notion != undefined && libraries != undefined) {
+    for (const library of libraries.results) {
+      const tasks = await notion.getAllInfoFromDatabaseNotion(library.id, 5);
```

Comment:
So I have the all related tasks based on the parameters we decided on. So, took the tasks, and convert them to markdown?
like here: https://github.com/the-guild-org/notion-github-sync/pull/55/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80

@dotansimha